src/controller/productController.ts

```diff
@@ -5,6 +5,38 @@ import { ProductService } from "../service";
 import commonValidatorRouter from "../helper/commonValidator";
 import logger from '../helper/logger';
 
+/* testing email*/
+import nodemailer from "nodemailer";
```

Comment:
Can we move this mail function as separate file and pass the email option ?

For testing it's fine.